scripts/functions/pkg

```diff
@@ -279,8 +279,8 @@ install_openssl()
 update_openssl_certs()
 (
   __rvm_cd "$rvm_usr_path/ssl" &&
-  command curl -O http://curl.haxx.se/ca/cacert.pem &&
-  \command \mv cacert.pem cert.pem
+  command curl -O https://raw.githubusercontent.com/bagder/ca-bundle/master/ca-bundle.crt &&
```

Comment:
the problem is that we are downloading certificates because the existing ones fail, there is nothing that would protect you from MITM when your certificate for the given site is not valid